CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- `stringtie` v2.2.1 added to detect splicing events. Not included in `fusion-report` or `fusionInspector` summaries
```

Comment:
```suggestion
- `stringtie` v2.2.1 added to detect splicing events. Not yet included in `fusion-report` or `fusionInspector` summaries
```
We probably work on that part

Reply:
The type of output is very different for stringtie: I don't think it can be processed like the other tools

Reply:
:disappointed: